### PR TITLE
Merge functions set_led

### PR DESCRIPTION
**Pull request by RaulTrombin**
*Needs Merge & Rebase:
- [ ] https://github.com/bluerobotics/navigator-rs/pull/33

Merge set_led methods to a single one.

Corrects the open milestone issue:
- [ ] https://github.com/bluerobotics/navigator-rs/issues/30

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -271,15 +271,10 @@ impl Led {
...
-
-        self.set_led(select, !state)
-    }
-
-    pub fn all_on(&mut self) {
+    pub fn set_led_all(&mut self, state: bool) {
         for pin in self.as_mut_array().iter_mut() {
-            pin.set_value(0).expect("Error: Set led value to 0");
+            pin.set_value((!state).into())
+                .unwrap_or_else(|_| panic!("Error: Set LED value to {}", state));
```
Comment (joaoantoniocardoso) [inline src/lib.rs:277]:
```Rust
panic!("Error: Set LED value to {state}");
```

File: src/lib.rs [hunk 2]
```diff
@@ -682,22 +681,14 @@ impl Navigator {
     ///
     /// nav.init();
     /// loop {
```
Comment (joaoantoniocardoso) [inline src/lib.rs:683]: isn't there a 'no_run' statement missing here?

**Review by joaoantoniocardoso — COMMENTED** [review]
Seems okay, just minor observations.

**Comment by patrickelectric** [comment]
@RaulTrombin please rebase.

**Review by patrickelectric — CHANGES_REQUESTED** [review]
needs rebase

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**